charts/osm-arc/templates/metrics-agent-deployment.yaml

```diff
@@ -1,3 +1,42 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: metrics-agent-account
+  namespace: {{ .Release.Namespace }}
+---
+kind: ClusterRole
```

Comment:
may make more sense to have rbac things added here instead: https://github.com/Azure/osm-azure/blob/main/charts/osm-arc/templates/metrics-rbac.yaml